code/__HELPERS/game.dm

```diff
@@ -471,3 +472,11 @@ proc/pollCandidates(var/Question, var/be_special_type, var/antag_age_check = 0,
 			candidates.Remove(G)
 
 	return candidates
+
+/proc/window_flash(client/C)
+    if(ismob(C))

```

Comment:
If the given mob lacks a client, won't the statement on line 480 runtime, as indexing `C.prefs` will attempt to reference something off of `null`?